src/dev/run_licenses_csv_report.js

```diff
@@ -101,16 +103,32 @@ export default function licensesCSVReport(grunt) {
 
       if (file) {
         writeFileSync(file, `${fields.join(',')}\n${csv}`);
-        grunt.log.writeln(`wrote to ${file}`);
+        log.success(`wrote to ${file}`);
       } else {
-        grunt.log.writeln(csv);
-        grunt.log.writeln('\nspecify "--csv [filepath]" to write the data to a specific file');
+        log.success(csv);
+        log.debug('\nspecify "--csv [filepath]" to write the data to a specific file');
       }
-
-      done();
     } catch (err) {
-      grunt.fail.fatal(err);
-      done(err);
+      process.exitCode = 1;
+      log.error(err);
+      return err;
```

Comment:
FYI, `run()` automatically catches errors and logs them and sets the error code so in the future we don't need to catch them like this.

Reply:
Pushed with 03af00c.  Tested by setting an invalid directory `--directory foo`